aviatrix-controller-build/variables.tf

```diff
@@ -0,0 +1,183 @@
+variable "mp_listings" {
+  type = map(string)
+  description = "Aviatrix Controller Marketplace listing IDs."
+  default = {
+    "PAID" = "ocid1.appcataloglisting.oc1..aaaaaaaawdptz5o42tuhg62defjimphzuck4fbrz4cilez2mle3bcs5t22wa"
+    "BYOL" = "ocid1.appcataloglisting.oc1..aaaaaaaavalzcaxbs4hwta4uefnkdh7fjgd3uuw7lwlk6pn2uvzglrn3zt2a"
+  }
+}
+
+variable "mp_byol_versions" {
+  type = map(string)
+  description = "Marketplace BYOL Version."
+  default = {
+    "6.3.0" = "6.3.1"
+  }
+}
+
+variable "mp_paid_versions" {
+  type = map(string)
+  description = "Marketplace PAID Version."
+  default = {
+    "6.3.0" = "6.3"
+  }
+}
+
+variable "mp_byol_listing_resource_id" {
+  type = map(string)
+  description = "Marketplace BYOL listing resource id."
+  default = {
+    "6.3.1" = "ocid1.image.oc1..aaaaaaaawyzirrayoeo5ivzkf57texkiu44ytnfifkmzry2xm5uu4otfk7cq"
+  }
+}
+
+variable "mp_paid_listing_resource_id" {
+  type = map(string)
+  description = "Marketplace PAID listing resource id."
+  default = {
+    "6.3" = "ocid1.image.oc1..aaaaaaaams7arhiyikfhpvo2kj35ky76csllxdnzijqw6ves4a5saaj63mva"
+  }
+}
+
+variable "product_version" {
+  type        = string
+  description = "Aviatrix Controller Version available in the Marketplace."
+  default     = "6.3.0"
+}
+
+variable "enabled" {
+  type        = bool
+  description = "Enable local values."
+  default     = true
+}
```

Comment:
Unclear what this is for. The variable and description is not clear of its purpose

Reply:
yes, I have remove this variable and add a new variable named: resource_count